src/content/en/fundamentals/performance/optimizing-content-efficiency/automating-image-optimization/index.md

```diff
@@ -0,0 +1,1615 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: Image formats!
+
+{# wf_updated_on: 2017-09-25 #}
+{# wf_published_on: 2017-09-25 #}
+
+# Automating Image Optimization {: .page-title }
+
+{% include "web/_shared/contributors/addyosmani.html" %}
+
+### <a id="the-tldr" href="#the-tldr">The tl;dr</a>
+
+**We should all be automating our image compression.**
+
+In 2017, image optimization should be automated. It's easy to forget, best practices change, and content that doesn't go through a build pipeline can easily slip.
```

Comment:
The style guide suggests [line lengths of 80 chars](https://developers.google.com/style/html-formatting#line-length).